azkaban-common/src/main/java/azkaban/executor/MysqlNamedLock.java

```diff
@@ -0,0 +1,35 @@
+package azkaban.executor;
+
+import azkaban.db.DatabaseTransOperator;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import javax.inject.Singleton;
+import org.apache.commons.dbutils.ResultSetHandler;
+
+/**
+ *  Util class for locking using named lock in mysql
+ */
+
+@Singleton
+public class MysqlNamedLock implements ResultSetHandler<Boolean> {
+  private String getLockTemplate = "SELECT GET_LOCK('%s', %s)";
+  private String releaseLockTemplate = "SELECT RELEASE_LOCK('%s')";
+
+  public boolean getLock(DatabaseTransOperator transOperator, String lockName, int lockTimeoutInSeconds) throws SQLException {
+    String getLockStatement = String.format(getLockTemplate, lockName, lockTimeoutInSeconds);
```

Comment:
Better to keep check on lockName and lockTimeoutInSeconds. There fields can not be null or <0 respectively.